mozilla-lockwise/lockwise-ios
# Ios 13 ui

Fixes [1099](https://github.com/mozilla-lockwise/lockwise-ios/issues/1099)
Fixes [1097](https://github.com/mozilla-lockwise/lockwise-ios/issues/1097)
Fixes [1113](https://github.com/mozilla-lockwise/lockwise-ios/issues/1113)

- Update codebase to newest version of Swift(5.1.2) and Xcode(11.2.1) and get everything functioning on iOS 13
- Fix navigation bar color and size issues caused by changes in iOS 13 on both iPhone and iPad
- Fix issue where logins do not appear on first launch due to iOS 13 change where `traitCollectionDidChange` is no longer called on initial view layout
- Fix all UI tests failing due to iOS 13 Safari updates

## Testing and Review Notes

- Launch app on both iOS and iPad
- Verify that header is rendering correctly on the Item List page and Item Detail page (screenshots below). Navigation bar background color should be purple and navigation title text should be small on Item List page and large on Item Detail page.  
- Verify login credential table view cells display on initial launch on all devices


## Screenshots or Videos

![ItemListView](https://user-images.githubusercontent.com/15383688/67965183-2fc4c880-fbc7-11e9-94ce-c9e6154df6b6.png)
![ItemDetailView](https://user-images.githubusercontent.com/15383688/67965193-33f0e600-fbc7-11e9-80e1-cd661c9cc0c2.png)
